setup.py

```diff
@@ -139,7 +138,7 @@ def compiler_test(
         if compiler.compiler_type == 'msvc':
             err.close()
             os.dup2(olderr, sys.stderr.fileno())
-            with open('err.txt', 'r') as err_file:
+            with open('err.txt') as err_file:
```

Comment:
Why was the `'r'` removed here?

Reply:
According to the Python standard, `'r'` is the default value for the mode. And therefore removed by `pyupgrade`